core/src/main/java/nl/nn/adapterframework/util/StreamUtil.java

```diff
@@ -39,7 +41,8 @@
  */
 public class StreamUtil {
 	public static final String DEFAULT_INPUT_STREAM_ENCODING="UTF-8";
-
+	public static final Charset DEFAULT_CHARSET = Charsets.UTF_8;
```

Comment:
Deze moeten toch altijd gelijk zijn aan elkaar?

Reply:
Maar een Charset is een Charset, en een String een String. Op de String kan je nog steeds een UnsupportedEncodingException krijgen, die je dan telkens moet afvangen